docs/Changelog.md

```diff
@@ -7285,29 +7285,29 @@ This version of the operator has been available since version 7 of the default O
 <dd>Number of neurons in the hidden layer</dd>
 </dl>
 
-#### Inputs (3 - 6)
+#### Inputs (5 - 6)
```

Comment:
This should not be changed. The two differentiable inputs should also be `Optional`.